helpers/sql-obfuscation/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History: opentelemetry-helpers-sql-obfuscation
 
+### v0.2.0 / 2024-09-10
+
+* BREAKING CHANGE: Return message when sql is over the obfuscation limit
+
+* FIXED: Return message when sql is over the obfuscation limit
```

Comment:
@arielvalentin  - Is this too wordy for a changelog entry? I have another option that's a little more concise in another comment below. I think these should be edited regardless, since the same message is listed twice.

Whatever we pick, I'll apply to the other changelogs.

```suggestion
* BREAKING CHANGE: Return a message when sql is over the obfuscation limit. Fixes a bug that sent un-obfuscated statements when sql was over the obfuscation limit and the statement had prepended comments.
```